cbdr/cortex
# Accordion Group Widget (EM-1322: Subscription FAQ)

## Purpose:
This PR implements Accordion Group Legacy Widget data storage, and retrieval via API

## JIRA:
https://cb-content-enablement.atlassian.net/browse/EM-1322

## Steps to Take On Prod
Prop up data detailed in EM-1322 in Webpage editor

## Changes:
* Changes to setup
  * N/A

* Architectural changes
  * N/A

* Migrations
  * Yes! :Thumb

* Library changes
  * N/A

* Side effects
  * N/A

## Screenshots
* Before
N/A

* After
N/A

## QA Links:
http://web.cortex-4.development.c66.me/

## How to Verify These Changes
* Specific pages to visit
  * Visit Webpage editor, ensure Accordion Group Widget shows up in Edit/New interface

* Steps to take
  * Ensure Widget saved
  * Ensure Accordion Group Widget is returned in API as JSON and Yaml

* Responsive considerations
  * N/A

## Relevant PRs/Dependencies:
https://github.com/cbdr/employer/pull/858
https://github.com/cortex-cms/cortex-snippets-client-ruby/pull/31
https://github.com/cb-talent-development/employer-style-base/pull/96

## Additional Information
N/A
